packages/worker/src/workflow/bundler.ts

```diff
@@ -315,6 +320,12 @@ export interface BundleOptions {
    * > NOTE: This is an advanced option that should be used with care.
    */
   ignoreModules?: string[];
+
+  /**
+   * Before we bundle the Workflow code with Webpack, we call `configureWebpack`, passing the Webpack
+   * {@link https://webpack.js.org/configuration/ | configuration} object so you can modify it.
+   */
+  configureWebpack?: (config: webpack.Configuration) => webpack.Configuration;
```

Comment:
I'd just make this return `void` and explicitly document that the config should be mutated by this function.

Reply:
I think `=> void` and "so you can modify it." suffices?

Reply:
I disagree on this one.

It should be possible to do something such as:

```
function myCustomizer(input: Webpack.Configuration) {
    return {
         ...input,
         someExtraField: "whatever",
    }
}
```
That pattern actually creates a new object.

Similarly, there are some utility packages that exists to help with customizing an existing webpack config (that can sometime be a non-trivial task). Take [this one](https://www.npmjs.com/package/webpack-merge) for example. Some of these packages actually returns a _new_ object, rather than modify the existing object.

Reply:
I prefer the flexibility of returning—you can mutate & return, or create new—will change back